### PR TITLE
[PBF-448] Fix for returning current time from getCurrentTime function

### PR DESCRIPTION
**Pull request by nagavenivn**

### CODE REVIEW

File: src/osmf/js/osmf_flash.js [hunk 1]
```diff
@@ -391,6 +391,7 @@ require("../../../html5-common/js/utils/constants.js");
      */
     this.getCurrentTime = function() {
       this.callToFlash("getCurrentTime");
+      return currentTime;
```
Comment (angieisrani) [inline src/osmf/js/osmf_flash.js]: how does this get set?

Comment (nagavenivn) [inline src/osmf/js/osmf_flash.js]: CURRENT_TIME event is dispatched from AS code to osmf_flash.js  and it gets updated in onCallback = _.bind(function(data)

Comment (angieisrani) [inline src/osmf/js/osmf_flash.js]: so when the return happens it's already updated?  Does the callback get called synchronously?

Comment (nagavenivn) [inline src/osmf/js/osmf_flash.js]: Yes, as this.callToFlash("getCurrentTime"); will call the function in AS which dispatches the event to JS and then return happens from the this.getCurrentTime

**Comment by angieisrani** [comment]
LGTM

**Pull request merged**